### PR TITLE
fix(alert, date-picker-month-header, input-date-picker, loader, panel): adds RTL equivalent CSS

### PR DESCRIPTION
**Pull request by Elijbet**
**Related Issue:** #3668

## Summary

Refactors to add support for RTL.

### CODE REVIEW

File: src/components/loader/loader.stories.ts [hunk 1]
```diff
@@ -31,7 +31,7 @@ export const inline_NoTest = (): string => html`
 </div>
 `;
 
-export const customTheme_NoTest = (): string => html`
+export const customThemeAndRTL_NoTest = (): string => html`
```
Comment (jcfranco) [inline src/components/loader/loader.stories.ts:34]: This story seems unrelated to `dir` (custom styling + RTL). 

I notice that the simple test is skipped for tests. @eriklharper @benelan can you confirm if this is intentional? If so, is it because the story's animation causes instability? 

If this is not being tested, I think we could skip testing it in RTL for the same reasons. At least until we switch to Chromatic.

File: src/components/date-picker-month-header/date-picker-month-header.scss [hunk 2]
```diff
@@ -117,7 +117,8 @@
 }
 
 .year--suffix {
-  @apply w-16 text-left;
+  @apply w-16
+  text-start;
```
Comment (jcfranco) [inline src/components/date-picker-month-header/date-picker-month-header.scss:121]: According to https://github.com/Esri/calcite-components/pull/5344, this needs to stay the same. cc @anveshmekala

Comment (Elijbet) [reply src/components/date-picker-month-header/date-picker-month-header.scss:121]: Thank you! I missed that.

File: src/components/loader/loader.stories.ts (hunk 1 above)
Comment (benelan) [reply src/components/loader/loader.stories.ts:34]: We shouldn't be taking any snapshots of Loader because every time Screener runs, the Loader position changes due to the animation. In Screener's UI we ignored the component, so every time there were visual regressions it wouldn't show up as a change. But it still took the snapshot every time, which was a waste.

Chromatic may have a better way of handling animated components, I'll look into that once we migrate.

Comment (Elijbet) [reply src/components/loader/loader.stories.ts:34]: That's a great recap, thanks for the info. I'll remove the RTL test and check back with you when we bring in Chromatic to address it then.

**Review by jcfranco — APPROVED** [review]
🎉
